packages/uxpin-merge-cli/src/steps/serialization/component/implementation/typescript/component/getComponentDocUrl.ts

```diff
@@ -0,0 +1,13 @@
+import * as ts from 'typescript';
+import { getUXpinDocUrlComment } from '../comments/getUXpinDocUrlComment';
+import { ComponentDeclaration } from './getPropsTypeAndDefaultProps';
+
+export function getComponentDocUrl(component:ComponentDeclaration):string | undefined {
+  const componentDocUrl:ts.JSDocTag | undefined = getUXpinDocUrlComment(component);
+
+  if (!componentDocUrl?.comment) {
+    return;
+  }
+
+  return componentDocUrl.comment;
```

Comment:
Maybe this is enough? 🤔 
```suggestion
  return componentDocUrl?.comment;
```